angel_system/ptg_eval/activity_classification/compute_scores.py

```diff
@@ -2,23 +2,23 @@
 from sklearn.metrics import average_precision_score
 import numpy as np
 
-log = logging.getLogger("ptg_eval")
+log = logging.getLogger("ptg_eval_activity")
 
 
 class EvalMetrics():
```

Comment:
Do we have plans to make this class more complicated? Wondering if it makes more sense recasting just as a function.

Reply:
Yeah I think there is/was intention to add more metrics